### ISSUE TITLE
How to get the exposure time of custom data?

### ISSUE THREAD

**Issue by nnop** [WU-CVGL/BAD-Gaussians#19]
Hi, thanks for the great work.
I have a collection of blurry images without exposure time.
Can I use the method on those images?

**Comment by LingzheZhao** [comment]
Hi, thanks for your interest in our work!

TLDR; Yes, you can. You don't need the exposure times with our method.

BAD-Gaussians will jointly optimize camera trajectory for every single image during the training of the 3DGS scene, given blurry images and SfM (e.g. COLMAP) outputs (estimated camera poses and point cloud from the blurry images).

________________________________
From: nnop ***@***.***>
Sent: Tuesday, August 27, 2024 8:31:27 PM
To: WU-CVGL/BAD-Gaussians ***@***.***>
Cc: Subscribed ***@***.***>
Subject: [WU-CVGL/BAD-Gaussians] How to get the exposure time of custom data? (Issue #19)


Hi, thanks for the great work.
I have a collection of blurry images without exposure time.
Can I use the method on those images?

―

**Comment by nnop** [comment]
Thanks. I'll have a try.
